src/gluonts/distribution/piecewise_linear.py

```diff
@@ -336,15 +336,7 @@ def __init__(self, num_pieces: int) -> None:
     @classmethod
     def domain_map(cls, F, gamma, slopes, knot_spacings):
         # slopes of the pieces are non-negative
-        thresh = F.zeros_like(slopes) + 20.0
-        I_grt_thresh = F.broadcast_greater_equal(slopes, thresh)
-        I_sml_thresh = F.broadcast_lesser(slopes, thresh)
-        slopes_proj = (
-            I_sml_thresh
-            * F.Activation(data=(I_sml_thresh * slopes), act_type="softrelu")
-            + I_grt_thresh * slopes
-        )
-        # slopes = F.Activation(slopes, 'softrelu')
```

Comment:
I'm not sure what all this logic was needed for: I checked and softrelu seems to behave fine with inputs much larger than 20 (when it's essentially the identity function). Maybe this was added in a tentative of fixing all other issues?

Reply:
Yes, this is outdated. There used to be an mxnet bug but the fix should already be in 1.4 (https://github.com/apache/incubator-mxnet/pull/6834/files).